ckanext/canada/ati.py

```
@@ -55,56 +66,82 @@ def command(self):
-    """
-    for dataset_type in DATASET_TYPES:
-        result = lc.action.package_search(
-            q="type:%s owner_org:%s" % (dataset_type, org['id']),
-            rows=1000)['results']
-        resource_id = result[0]['resources'][0]['id']
-        offset = 0
-        while True:
-            rval = lc.action.datastore_search(resource_id=resource_id,
-                limit=BATCH_SIZE, offset=offset)
-            records = rval['records']
-            if not records:
-                break
-            yield records
-            offset += len(records)
+        if csv_files:
+            count = {}
+            for csv_file in csv_files:
+                try:
+                    print csv_file + ':'
+                    count[csv_file] = {}
+                    for org_recs in csv_data_batch(csv_file, TARGET_DATASET):
+                        org_id = org_recs.keys()[0]
+                        if org_id not in count[csv_file]:
+                            count[csv_file][org_id] = 0
+                        org_detail = lc.action.organization_show(id=org_id)
+                        records = org_recs[org_id]
+                        _update_records(records, org_detail, conn)
+                        count[csv_file][org_id] += len(records)
+                    for k, v in count[csv_file].iteritems():
+                        print "    {0:s} {1}".format(k, v)
+                except _csvError as e:
+                    logging.error('On {0:s}, encountered: {1:s}'.format(
+                        csv_file,
+                        e.message))
+                except AssertionError as e:
+                    logging.warning('On {0:s}, encountered: {1:s}'.format(
+                        csv_file,
+                        e.message))
+                except IOError as e:
+                    logging.error('On {0:s}, encountered: {1:s}'.format(
+                        csv_file,
+                        e.strerror))
```

This swallows the tracebacks and only keeps the error message. I'm ok with letting the traceback rise normally and cancel the whole process. We don't want to ignore errors here (and we certainly want all the information necessary to fix them).  Please remove the except: clauses here.